kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/Console.java

```diff
@@ -386,7 +386,7 @@ private synchronized void initHTTPService() throws NamespaceException, ServletEx
         this.httpService.registerServlet(CONSOLE_RESOURCE_PATH, new SendStatusServlet(404), null, resourceContext);
 
         this.httpService.registerServlet(PASSWORD_AUTH_PATH,
-                new GwtPasswordAuthenticationServiceImpl(this.authMgr, CONSOLE_PATH), null, sessionContext);
```

Comment:
I think this should be left to CONSOLE_PATH, for the reasons we discussed in the past.